README.md

```diff
@@ -32,7 +32,7 @@ simpler and better-suited to the needs of designers and cartographers.
     open('tile.png', 'w').write(bytes)
 
 
-##Quickstart
+## Quickstart
 
 To make sure TileStache is working start the development server:
 
```

Comment:
Can we add instructions on how to run the container and maybe the expected behaviour?
Something like this
```
docker run -it --rm --net=host ktile:latest
```
Then go to http://localhost:8080/osm/0/0/0.png